crypto/BRCryptoNetwork.c

```diff
@@ -814,15 +815,16 @@ cryptoNetworkInstallBuiltins (BRCryptoCount *networksCount) {
         }
 
         // Create the Network Fees
+        BRCryptoUnit feeUnit = cryptoNetworkGetUnitAsBase (network, cryptoNetworkGetCurrency(network));
```

Comment:
Is feeUnit leaked? Is `cryptoNetworkGetCurrency()` result leaked?

Reply:
Isn't the cryptoNetworkGetCurrency result leaked?

Reply:
Fixed that.  And another once over.

Reply:
The joys of memory management; looks good!